packages/frontend/src/app/app.module.ts

```diff
@@ -1,30 +1,34 @@
 import {BrowserModule} from '@angular/platform-browser';
 import {NgModule} from '@angular/core';
-
+import {HttpClient, HttpClientModule} from '@angular/common/http';
 import {TranslateLoader, TranslateModule} from '@ngx-translate/core';
 import {TranslateHttpLoader} from '@ngx-translate/http-loader';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
 
 import {AppRoutingModule} from './app-routing.module';
 import {AppComponent} from './app.component';
-import {HttpClient, HttpClientModule} from '@angular/common/http';
+import {NavbarModule} from './shared/navbar/navbar.module';
+import {LanguageService} from './shared/language.service';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
   ],
   imports: [
     BrowserModule,
     AppRoutingModule,
     HttpClientModule,
+    NavbarModule,
     TranslateModule.forRoot({
       loader: {
         provide: TranslateLoader,
         useFactory: TranslationLoaderFactory,
         deps: [HttpClient]
       }
-    })
+    }),
+    BrowserAnimationsModule,
   ],
-  providers: [],
+  providers: [LanguageService],
```

Comment:
I think its unnecessary because You set {provideIn: root} in service annotation.